ui/src/app/components/copy-modal.spec.tsx

```diff
@@ -58,72 +40,107 @@ interface TestWorkspace {
   accessTierShortName: string;
 }
 
-function simulateWorkspaceSelection(
-  wrapper: ReactWrapper,
-  workspace: TestWorkspace
-) {
-  simulateSelect(wrapper, Select, workspace.name);
-}
+const openWorkspaceSelection = async () => {
```

Comment:
I like these methods, does it make sense to move them to a common place for other tests to use?

Reply:
thanks - but I think in this case no, because they're pretty specific to the functionality of this component